css/auth.css

```
@@ -71,17 +73,44 @@
   gap:1rem;
   font-size:1.4rem;
   font-weight:500;
-  color:#1F2937;
+  color:var(--color-secondary800);
   justify-content: center;
   align-items: center;
 }
 .login_area .signup_desc .signup_link{
-  color:#3692FF;
+  color:var(--color-primary100);
   font-size:1.4rem;
   font-weight:500;
 }
 
 .login_area .signup_desc .signup_link:hover,
 .login_area .signup_desc .signup_link:focus{
   text-decoration:underline;
+}
+
+.inp_box input[type="password"]#password_confirm{
+  background-image:url(../img/btn_visibility_on_24px.svg);
+}
+
+
+
+@media screen and (max-width: 1199px){
+
+  /* 테블릿 영역 */
```

💊 제안
해당 주석은 미디어 쿼리 위에 있는게 더 적절할 것 같아요~

```suggestion
/* 테블릿 영역 */
@media screen and (max-width: 1199px){
```